selfdrive/car/honda/carstate.py

```diff
@@ -192,7 +192,7 @@ def update(self, cp, cp_cam, cp_body):
       ret.standstill = not cp.vl["STANDSTILL"]["WHEELS_MOVING"]
       ret.doorOpen = bool(cp.vl["SCM_BUTTONS"]["DRIVERS_DOOR_OPEN"])
     else:
-      ret.standstill = not cp.vl["STANDSTILL"]["WHEELS_MOVING"]
+      ret.standstill = cp.vl["ENGINE_DATA"]["XMISSION_SPEED"] < 0.1
```

Comment:
This is not a problem on the FREED and HRV?

Reply:
Oh I think it is as well, forgot where someone reported that to me. However even the XMISSION_SPEED signal has some crazy high minimum speed it reports (or it's gated on brake pressed), as I was able to creep at at least a few mph and the signal was still 0. I want to look for a new bit for the standstill, just haven't had time

Reply:
Used a script to find any bits matching CS.vEgo < (or greater than) 0.1 m/s, however nothing matched as close as using the `XMISSION_SPEED` which was the most accurate and not noisy at all compared to `STANDSTILL->WHEELS_MOVING`. However it does drop to zero at 1 kph, so you can engage up to half a mph with brake pressed.